### PR TITLE
5.3.0 final history

### PR DESCRIPTION
**Pull request by sbesson**
This PR should include the last set of changes post 5.3.0-m3 into the version history.

### CODE REVIEW

**Comment by sbesson** [comment]
Flagged by @hflynn, 4a6625a reviews and updates all remaining references to Bio-Formats 5.2 including the Javadoc API links. This change led to a number of broken links in  https://ci.openmicroscopy.org/job/BIOFORMATS-DEV-merge-docs/512/ that should be fixed in the following commits especially:
- updated links to Javadoc for decoupled components (`ome-common` and `ome-xml`)
- updated links to Javadoc methods. It seems Java 8 generated Javadoc is now using hyphens instead of bracket i.e. `http://base_url/package/class#method-arguments-`

**Comment by sbesson** [comment]
With the last commit, all hyperlinks should be fixed. See in particular the following staging pages for review:

- https://www.openmicroscopy.org/site/support/bio-formats5.3-staging/about/whats-new.html for the version history
- https://www.openmicroscopy.org/site/support/bio-formats5.3-staging/formats/zeiss-czi.html for the note about the jxrlib platform /cc @melissalinkert @chris-allan

File: docs/sphinx/developers/components.txt [hunk 1]
```diff
@@ -254,3 +255,10 @@ which are required at compile time but cannot be included in the build system
 (usually due to licensing issues).  The build succeeds since required class
 names are present with the correct method signatures; the end user is then
 expected to replace the stub .jar files at runtime.
+
+.. _jxrlib:
+
+`JXRlib <https://github.com/glencoesoftware/jxrlib>`__:
+
+This component contains the Java bindings for jxrlib an open source
```
Comment (simleo) [inline docs/sphinx/developers/components.txt]: Add a comma after "jxrlib"

File: docs/sphinx/index.txt [hunk 2]
```diff
@@ -20,8 +20,8 @@ supported by Bio-Formats.
     - :doc:`developers/index`
     - :doc:`formats/index`
 
-    The version *5.2* releases use the *June 2016* release of the
-    :model_doc:`OME-Model <>`.
+    Bio-Formats |release| use the *June 2016* release of the
```
Comment (simleo) [inline docs/sphinx/index.txt]: "use" -> "uses"

**Comment by hflynn** [comment]
You haven't updated the version number and date at the top

**Comment by hflynn** [comment]
Aside from the corrections Simone has already suggested, I can't see any issues besides needing to update the version number and date as I mentioned above.

**Comment by hflynn** [comment]
👍 Should be good to merge now

**Pull request merged**